program/src/processor.rs

```diff
@@ -2577,7 +2613,7 @@ impl Processor {
             if init_health > ZERO_I80F48 {
                 liqee_ma.being_liquidated = false;
                 msg!("Account init_health above zero.");
-                return Ok(());
+                return Err(throw_err!(MangoErrorCode::NotLiquidatable));
```

Comment:
This will revert changes and we need to keep the change to liqee_ma.being_liquidated